code/datums/components/crafting/atmospheric.dm

```diff
@@ -17,6 +17,16 @@
 	time = 0.5 SECONDS
 	category = CAT_ATMOSPHERIC
 
+/datum/crafting_recipe/igniter
+	name = "Igniter"
+	result = /obj/machinery/igniter
+	reqs = list(
+		/obj/item/stack/sheet/iron = 5,
+		/obj/item/assembly/igniter = 1,
```

Comment:
what if i craft it with a condenser?

Reply:
but that doesn't make sense cause it's meant to ignite gases not cool them down. I have an idea for the next PR where i create a stationary condenser which will use just that but for this, i don't understand how it would logically make sense

Reply:
Nice i figured it out